MH-FiorellaTheme5/source/layouts/layout.erb

```diff
@@ -3,21 +3,35 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
As a suggestion... we should not be using `user-scalable=no` because of problems with accessibility, there may be users that needs to zoom in to read content in the whole page.